staging/konvoyconfig/values.yaml

```diff
@@ -25,3 +25,7 @@ config:
 # It's possible to set the full Dex issuer URL, however, this is usually not
 # necessary. Only use this if you know what you are doing.
 # clusterIssuerURL: https://name1.example.com/custom/dex
+
+# caCertificate takes the name of the secret that holds the CA bundle for the
+# custom certificate.
+# caCertificate: custom-cert
```

Comment:
We've agreed with @GoelDeepak that this key `caCertificate` will be renamed to `caSecretName` to better represent the expected value, which is a reference to secret rather than PEM content.